src/app/tests/suites/certification/Test_TC_S_2_6.yaml

```diff
@@ -52,114 +105,243 @@ tests:
       PICS: S.S.C03.Rsp
       verification: |
           ./chip-tool scenesmanagement remove-all-scenes 0x0000 2 1 --commissioner-name beta
+
+          Verify the RemoveAllScenesResponse with following fields:
+          Status is SUCCESS
+          Group ID is 0x0000  on the TH(Chip-tool) log and below is the sample log provided for the raspi platform:
+
+          [1700826575.191275][15971:15973] CHIP:DMG: Received Command Response Data, Endpoint=1 Cluster=0x0000_0062 Command=0x0000_0003
+          [1700826575.191321][15971:15973] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0062 Command 0x0000_0003
+          [1700826575.191938][15971:15973] CHIP:TOO:   RemoveAllScenesResponse: {
+          [1700826575.191974][15971:15973] CHIP:TOO:     status: 0
+          [1700826575.191985][15971:15973] CHIP:TOO:     groupID: 0
+          [1700826575.191995][15971:15973] CHIP:TOO:    }
+
           ./chip-tool scenesmanagement remove-all-scenes 0x0000 3 1 --commissioner-name gamma
```

Comment:
This can be done in a follow up: testspec should be updated TH3 doing RemoveAllScenes as it's own standalone step 1c.

Reply:
Opened issue: https://github.com/project-chip/matter-test-scripts/issues/160